tests/OpenCFP/Http/Controller/Admin/TalksControllerTests.php

```diff
@@ -0,0 +1,184 @@
+<?php
+namespace OpenCFP\Tests\Http\Controller\Admin;
+
+use Mockery as m;
+use Symfony\Component\HttpFoundation\Session\Session;
+use Symfony\Component\HttpFoundation\Session\Storage\MockFileSessionStorage;
+use OpenCFP\Application;
+use OpenCFP\Environment;
+
+class AdminTalkControllerTest extends \PHPUnit_Framework_TestCase
+{
+    public $app;
+
+    public function setup()
```

Comment:
@chartjes 

Visibility of parent is `protected`, should the visibility be the same here?